scripts/supertonic/dump_inputs.py

```
@@ -0,0 +1,335 @@
+        # Load voice style
+        style = load_voice_style(args.voice_style, verbose=True)
+
+        # Process sentences
+        print(f"\nProcessing {args.n_test} sentence(s)...")
+        for n in range(args.n_test):
+            print(f"\n[{n+1}/{args.n_test}]")
+
+            if args.batch:
+                wav, duration = tts.batch(args.text, args.lang, style, args.total_step, args.speed)
+            else:
+                wav, duration = tts(args.text[0], args.lang[0], style, args.total_step, args.speed)
```

Ruff (0.14.14)

[warning] Unpacked variable `wav` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)